React/Fabric/Mounting/RCTComponentViewRegistry.mm

```diff
@@ -83,7 +83,11 @@ - (void)preallocateViewComponents
       @"RCTComponentViewRegistry: Attempt to dequeue already registered component.");
 
   auto componentViewDescriptor = [self _dequeueComponentViewWithComponentHandle:componentHandle];
+#if !TARGET_OS_OSX // [macOS]
   componentViewDescriptor.view.tag = tag;
+#else // [macOS
+  componentViewDescriptor.tag = tag;
+#endif // macOS]
```

Comment:
Have we not yet set a view yet on macOS, so instead we're directly setting the tag, presumably to a default value of 0 everywhere? Do we have an issue tracking setting it on the actual view? (either internally at Meta or publicly on our GH project)

Reply:
In the upcoming TouchHandler changes, this will be set to correct react tag.